app/helpers/application_helper.rb

```diff
@@ -127,6 +127,11 @@ def needs_login
     logged_in? ? '' : 'needs_login'
   end
 
+  def api_pagination(response)
+    numbers = (1..response['total_entries']).to_a
+    will_paginate(numbers.paginate(page: response['current_page'], per_page: response['per_page']))
```

Comment:
This one looks like a hacky solution. As per the rails convention, it seems, we should show `first`, `previous`, `next` and `last` links https://github.com/davidcelis/api-pagination